skyportal/handlers/api/comment.py

```diff
@@ -118,47 +113,14 @@ def post(self):
             return self.error("Missing required field `obj_id`")
         comment_text = data.get("text")
 
-        # Ensure user/token has access to associated Obj
-        _ = Obj.get_if_readable_by(obj_id, self.current_user)
-        user_accessible_group_ids = [g.id for g in self.current_user.accessible_groups]
-        user_accessible_filter_ids = [
-            filtr.id
-            for g in self.current_user.accessible_groups
-            for filtr in g.filters
-            if g.filters is not None
-        ]
-        group_ids = [int(id) for id in data.pop("group_ids", user_accessible_group_ids)]
-        group_ids = set(group_ids).intersection(user_accessible_group_ids)
+        group_ids = data.pop('group_ids', None)
         if not group_ids:
-            return self.error(
-                f"Invalid group IDs field ({group_ids}): "
-                "You must provide one or more valid group IDs."
-            )
-
-        # Only post to groups source/candidate is actually associated with
```

Comment:
Is there any assurance that specified groups are actually associated with the source?

Reply:
Since `Obj`s are `read=public`, no need to check this anymore. But if one wanted to impose a condition like this, it could be done in `GroupComment.create`.

Reply:
That condition would be automatically enforced in `self.verify_permissions()`.

Reply:
I believe the reason we had added this check was that under certain conditions we were actually getting error responses for certain fetch queries. Hopefully we don't run into that again with the new access control infrastructure.

Reply:
Shouldn't be an issue because if a source exists, it will be fetchable by anyone. The only way this would error is if someone tries to fetch a nonexistent source. The only source info that is subject to non-public access controls are the attendant data (photometry, spectra, etc.).